commercetools/commercetools-sdk-java-importapi/src/main/java-generated/com/commercetools/importapi/models/importoperations/ImportOperationState.java

```diff
@@ -16,20 +16,20 @@
 public interface ImportOperationState {
 
     /**
-    	<p>This is an initial state of import resource and is currently unresolved.</p>
+    	<p>This is an initial state of import resource and is currently processing.</p>
 
     */
-    ImportOperationState UNRESOLVED = ImportOperationStateEnum.UNRESOLVED;
+    ImportOperationState PROCESSING = ImportOperationStateEnum.PROCESSING;
     /**
     	<p>The validation of the import resource failed.</p>
 
     */
     ImportOperationState VALIDATION_FAILED = ImportOperationStateEnum.VALIDATION_FAILED;
 
     enum ImportOperationStateEnum implements ImportOperationState {
-        UNRESOLVED("Unresolved"),
+        PROCESSING("processing"),
```

Comment:
breaking

Reply:
yep same, customers have knowledge about it